.gitignore

```diff
@@ -44,3 +44,7 @@ Thumbs.db
 
 *kubernetes/secrets/**
 *kubernetes.yaml
+
+# Testing
+cypress
```

Comment:
Is this a top level directory? It might be good to indicate it as a top level directory

Reply:
Thanks resolved.